src/jsifier.js

```diff
@@ -1,549 +1,561 @@
+      if (ident.endsWith('__sig') || ident.endsWith('__proxy') || ident.endsWith('__asm') || ident.endsWith('__inline') || ident.endsWith('__deps') || ident.endsWith('__postset')) {
+        return '';
+      }
+
+      // $ident's are special, we do not prefix them with a '_'.
+      if (ident[0] === '$') {
+        var finalName = ident.substr(1);
+      } else {
+        var finalName = '_' + ident;
+      }
+
+      // if the function was implemented in compiled code, we just need to export it so we can reach it from JS
+      if (finalName in IMPLEMENTED_FUNCTIONS) {
+        EXPORTED_FUNCTIONS[finalName] = 1;
+        // stop here: we don't need to add anything from our js libraries, not even deps, compiled code is on it
+        return '';
+      }
+
+      // Don't replace implemented functions with library ones (which can happen when we add dependencies).
+      // Note: We don't return the dependencies here. Be careful not to end up where this matters
+      if (finalName in Functions.implementedFunctions) return '';
+
+      var noExport = false;
+
+      if (allExternPrimitives.indexOf(ident) != -1) {
+        usedExternPrimitives[ident] = 1;
+        return;
+      } else if ((!LibraryManager.library.hasOwnProperty(ident) && !LibraryManager.library.hasOwnProperty(ident + '__inline')) || SIDE_MODULE) {
+        if (!(finalName in IMPLEMENTED_FUNCTIONS)) {
+          if (VERBOSE || ident.substr(0, 11) !== 'emscripten_') { // avoid warning on emscripten_* functions which are for internal usage anyhow
+            if (!LINKABLE) {
+              if (ERROR_ON_UNDEFINED_SYMBOLS) {
+                error('undefined symbol: ' + ident);
+                warnOnce('To disable errors for undefined symbols use `-s ERROR_ON_UNDEFINED_SYMBOLS=0`')
+              } else if (VERBOSE || WARN_ON_UNDEFINED_SYMBOLS) {
+                warn('undefined symbol: ' + ident);
+              }
+            }
+          }
+        }
+        if (!RELOCATABLE) {
+          // emit a stub that will fail at runtime
+          LibraryManager.library[ident] = new Function("err('missing function: " + ident + "'); abort(-1);");
+        } else {
+          var isGlobalAccessor = ident.startsWith('g$');
+          var realIdent = ident;
+          if (isGlobalAccessor) {
+            realIdent = realIdent.substr(2);
+          }
+
+          var target = (SIDE_MODULE ? 'parent' : '') + "Module['_" + realIdent + "']";
+          var assertion = '';
+          if (ASSERTIONS) {
+            var what = 'function';
+            if (isGlobalAccessor) {
+              what = 'global';
+            }
+            assertion += 'if (!' + target + ') abort("external ' + what + ' \'' + realIdent + '\' is missing. perhaps a side module was not linked in? if this function was expected to arrive from a system library, try to build the MAIN_MODULE with EMCC_FORCE_STDLIBS=1 in the environment");\n';

```

Comment:
Something seems to have gone wrong with the diff here.. maybe you changed the line endings to windows by accident?

Reply:
I think its because I forgot to do another merge from incoming before sending the review. Does it look ok now?